kore/src/Kore/Log/KoreLogOptions.hs

```diff
@@ -142,6 +154,19 @@ parseKoreLogType =
             <> Options.long "log"
             <> Options.help "Name of the log file"
 
+parseKoreLogFormat :: Parser KoreLogFormat
+parseKoreLogFormat = option formatReader info
+  where
+    formatReader = Options.maybeReader $ \case
+        "standard" -> Just Standard
+        "oneline" -> Just OneLine
+        _ -> Nothing
+    info =
+        mempty
```

Comment:
No need for `mempty` due to monoid identity law: `mempty <> x = x`

Reply:
Yeah, that's there just so that we can line up the stuff after with `<>` at the beginning. It makes the code tidier, easier to read and I believe we use this style in other places too.